src/Maestro/maestro-angular/src/app/page/build/build.component.html

```diff
@@ -50,15 +50,25 @@ <h4>{{getRepo(build)}} <small class="text-muted">{{build.azureDevOpsBuildNumber}
         <div class="flex-grow-1">
           <div class="float-right">
             <div>
-              <a class="btn btn-sm btn-primary" [href]="build | buildLink" target="_blank">
+              <a class="btn btn-sm btn-primary float-right mb-2" [href]="build | buildLink" target="_blank">
                 Go to build in Azure DevOps <fa-icon icon="external-link-alt"></fa-icon>
               </a>
             </div>
-            <div *ngIf="azDevInfo.mostRecentFailureLink">
-              <a class="btn btn-danger btn-sm float-right text-light" [href]="azDevInfo.mostRecentFailureLink"
-                target="_blank">
-                Latest Failed Build <fa-icon icon="external-link-alt"></fa-icon>
-              </a>
+            <div>
+              <span *ngIf="azDevInfo.mostRecentFailureLink">
+                <a class="btn btn-danger btn-sm float-right text-light ml-2" [href]="azDevInfo.mostRecentFailureLink"
+                  target="_blank">
+                  Latest Failed Build <fa-icon icon="external-link-alt"></fa-icon>
+                </a>
+              </span>
+              <ng-container *stateful="let azDevOnGoingInfo from azDevOnGoingBuildsInfo$; loadingTemplate: loadingBuildInfo; errorTemplate: errorBuildInfo;">
```

Comment:
The templates here don't seem correct.  That template is a 50 pixel tall spinning ring.  And the errorBuildInfo produces a bunch of weird information that has nothing to do with the ongoing build.